tripy/tripy/config.py

```diff
@@ -38,6 +38,8 @@
 enable_tensorrt_debug = os.environ.get("TRIPY_TRT_DEBUG_ENABLED", "0") == "1"
 tensorrt_debug_path = os.environ.get("TRIPY_TRT_DEBUG_PATH", os.path.join("/", "tripy", "tensorrt-dumps"))
 
+eager_cache = os.environ.get("TRIPY_EAGER_CACHE", "0") == "1"
```

Comment:
We will want to export this as a public API (i.e. `export.public_api` like below)

Reply:
done